Sources/ApolloCodegenLib/ApolloSchemaDownloadConfiguration.swift

```diff
@@ -54,16 +54,21 @@ public struct ApolloSchemaDownloadConfiguration {
     public var debugDescription: String {
       "\(key): \(value)"
     }
+
+    public init(key: String, value: String) {
+      self.key = key
+      self.value = value
+    }
   }
 
   /// How to download your schema. Supports the Apollo Registry and GraphQL Introspection methods.
-  let downloadMethod: DownloadMethod
+  public let downloadMethod: DownloadMethod
```

Comment:
There's no reason to keep these private; once you've created the struct you should be able to inspect the property values.

Reply:
True for as long as they remain `let`s - If they're `var`s we may or may not want to allow direct access, although we could handle that with a `private(set)`.